proto/dydxprotocol/ratelimit/tx.proto

```diff
@@ -5,15 +5,12 @@ option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/ratelimit/types
 
 import "cosmos/msg/v1/msg.proto";
 import "dydxprotocol/ratelimit/limit_params.proto";
+import "gogoproto/gogo.proto";
 
 // Msg defines the Msg service.
 service Msg {
   // SetLimitParams sets a `LimitParams` object in state.
   rpc SetLimitParams(MsgSetLimitParams) returns (MsgSetLimitParamsResponse);
-
-  // DeleteLimitParams removes a `LimitParams` object from state.
-  rpc DeleteLimitParams(MsgDeleteLimitParams)
```

Comment:
No long use `DeleteLimitParams`, since we use `SetLimitParams` for delete when `Limiters` is empty